scripts/download_image.py

```diff
@@ -277,18 +281,15 @@ def show_game_download_warning(workers: int,
     >>> show_game_download_warning(10, input_fn=lambda x: "n")
     True
     """
-    choice = input_fn(
-            'WARNING: You specified {} workers which is over limit {'
-            '} to '
-            'download images from game website. This may lead you to '
-            'get '
-            'banned. If you accept it enter Y or enter N to exit '
-            'script. '
-            'If not please define number of threads with -w '
-            'parameter. '
-            'If you want to skip this warning you can enter -y while '
-            'calling script.:'.format(workers,
-                                      GAME_WORKER_LIMIT)).lower()
+    warning = 'WARNING: You specified {} workers which is over limit {} to ' \
+              'download images from game website This may lead you to get ' \
+              'banned. If you accept it enter Y or enter N to exit script. ' \
+              'If not please define number of threads with -w parameter. If ' \
+              'you want to skip this warning you can enter -y while calling ' \
+              'script : '.format(
+        workers, GAME_WORKER_LIMIT)
+    warning = '\n'.join(warning[i:i + 92] for i in range(0, len(warning), 92))
```

Comment:
no, don't want this trick. write in top-level 
```python
WARNING_MESSAGE = """\
    WARNING: 
     ...
"""
```

textwrap.dedent(text) helps this.

Reply:
Changed with auto terminal size splitting:
```
    choice = input_fn('\n'.join(textwrap.wrap(
        textwrap.dedent(WARNING_MESSAGE.format(workers, GAME_WORKER_LIMIT)),
        shutil.get_terminal_size().columns))).lower()
```
WARNING: You specified 10 workers which is over limit 1 to download images from
game website This may lead you to get banned. If you accept it enter Y or enter
N to exit script. If not please define number of threads with -w parameter. If
you want to skip this warning you can enter -y while calling script :

Reply:
can you remove shutil.get_terminal_size() and tidy WARNING_MESSAGE
readable in source code too.

Reply:
an no, choice == 'y' may False